flask_nemo/__init__.py

```diff
@@ -499,6 +499,7 @@ def expose_ancestors_or_children(self, member, collection, lang=None):
         x = {
             "id": member.id,
             "label": str(member.get_label(lang)),
+            "is_resource": isinstance(member, ResourceCollection),
```

Comment:
Can you add a test for that ?

Reply:
A plain text checking that on collection route, an Edition has the Browse link :)